src/app/work-item/work-item-detail/work-item-detail.component.html

```diff
@@ -211,7 +211,9 @@ <h2 *ngIf="loggedIn" id="wi-detail-title-click" class="detail-title truncate">{{
         <!--Description-->
         <div class="separator col-md-12 col-sm-12 col-xs-12 detail-description-wrap">
           <div class="col-md-12 col-sm-12 col-xs-12 detail-description" *ngIf="loggedIn">
-            <div class="detail-desc-div-wrap" (click)="openDescription(false, true)">
+            <button class="view-switch-btn" *ngIf="descViewType=='markdown'" (click)="showHtml(desc.innerText)">View HTML</button>
```

Comment:
We can but doesn't have any use. Besides, the UX is not defined, it's just the functionality that we have implemented. Once the UX in place we'll have to rework on buttons and stuff so currently the experience is very basic and working model